src/locales/it.yaml

```diff
@@ -0,0 +1,360 @@
+          @.capitalize:app.general.simple_form.msg.confirm Quest'azione spegnerà
+          il tuo sistema host.
+      error:
+        invalid_url: URL Invalido
+        max: Massimo %{max}
+        min: Minimo %{min}
+        min_or_0: Minimo %{min} o 0
+        required: Richiesto
+    tooltip:
+      estop: Fermata d'emergenza
+      reload_klipper_config: Ricarica la configurazione di klipper.
+      reload_klipper_config_and_restart: Ricarica la configurazione di klipper e riavvia il Firmware.
+      restart_klipper: Riavvia il servizio di klipper.
+    error:
+      app_warnings_found: '%{appName} trovati warning.'
+      app_setup_link: >-
+        I requisiti per impostare Fluidd li trovi <a target="_blank"
+        href="%{url}">qua.</a>
+      failed_components: >-
+        Moonraker ha problemi con alcuni plugin, perfavore controlla i tuoi log,
+        aggiorna la tua configurazione e riavvia Moonraker.
+      components_config: >-
+        La configurazione dei plugin di Moonraker la trovi <a target="_blank"
+        href="%{url}">qua.</a>
+  socket:
+    msg:
+      connecting: Connettendomi a Moonraker...
+      no_connection: >-
+        Nessuna connessione a Moonraker. Perfavore controllane lo stato e / o
+        prova a ricaricare.
+  version:
+    title: Aggiornamenti
+    btn:
+      update: Aggiorna
+      finish: Fine
+      check_for_updates: Controlla gli Aggiornamenti
+    label:
+      dirty: SPORCO
+      invalid: INVALIDO
+      up_to_date: AGGIORNATO
+      commit_history: Cronologia Commit
+      commits_on: Abilita Commit
+      committed: Committed
+      package_list: Lista Pacchetti
+    status:
+      updating: Aggiornando...
+      finished: Aggiornamenti finiti.
+    tooltip:
+      dirty: indica un head staccato, non è un master o ha un origine invalida
+      invalid: indica cambiamenti locali alla repo
+      commit_history: Cronologia Commit
+      packages: Pacchetti
+      release_notes: Note di Rilascio
+  console:
+    label:
+      hide_temp_waits: Nascondi Temperature di Riscaldamento
+    placeholder:
+      command: >-
+        'tab' per l'autocompletamento, 'help' per i comandi 'freccia su / giu'
+        per la cronologia
+  tool:
+    btn:
+      home_x: X
+      home_y: 'Y'
+    tooltip:
+      home_xy: Home XY
+      home_z: Home Z
+      extruder_disabled: >-
+        Estrusore disattivato, temperatura inferiore a min_extrude_temp
+        (%{min}<small>°C</small>)
+  bedmesh:
+    label:
+      profile_name: Nome Profilo
+      remove_profile: 'Rimuovi il profilo: %{name}'
+    msg:
+      not_found: Nessuna mesh per il letto trovata.
+      hint: >-
+        Se lo salvi con un nome diverso da %{name},  puoi anche rimuovere il
+        profilo: %{name}
+    tooltip:
+      calibrate: Inizia una nuova calibratura, salvato come profilo 'default'
+      delete: Elimina il profilo. RIAVVIERÀ la tua stampante
+      save: Salva la calibrazione in printer.cfg. RIAVVIERÀ la tua stampante
```

Comment:
Suggestion: `Salva la calibrazione in printer.cfg. Questa azione RIAVVIERÀ la tua stampante`